deepchecks/core/display.py

```diff
@@ -92,7 +92,9 @@ def _repr_html_(self):
 
             return TempSphinx()
 
-        if is_colab_env() and as_widget is True:
+        if is_kaggle_env():
```

Comment:
you need then change it to this

```
if is_kaggle_env() and is_databricks_env() and is_sagemaker_env():
    self.show_in_iframe(as_widget=as_widget, unique_id=unique_id, **kwargs)
```

Reply:
Do we have a mechanism to check if we're on sagemaker?

Reply:
plus, I think it is worth showing a warning, something like 
`this envs do not support ipywidgets (or have troubles with thems) therefore we will use iframe`

Reply:
sorry, did not see you prev comment, no we do not have